examples/text-generation/run_generation.py

```diff
@@ -222,6 +222,7 @@ def setup_parser(parser):
     )
 
     parser.add_argument("--fp8", action="store_true", help="Enable Quantization to fp8")
+    parser.add_argument("--int4", action="store_true", help="Enable Quantization to int4")
```

Comment:
is it just for int4 or also for uint ?
if also for uint i suggest to change the arg to something like "gptq_mode" which can get "i4","u4" values. or something similar